CONTRIBUTING.md

```diff
@@ -0,0 +1,37 @@
+# Contributing
+
+This doc explains how to get started contributing to [baikonur-oss](https://github.com/baikonur-oss) project.
+
+## Filing an Issue
+
+If you are trying to use baikonur-oss modules and run into some issues, please file an issue. We can look into the issue more easily if you describe in details as possible as you can. The following is example detailed descriptions:
+
+- Version of tools and modules
```

Comment:
PR templates to come?

Reply:
Yep. We should include a PR template in baikonur template repository. This is a temporary work.